frontend/src/premium/user/components/list/user-list-filter.vue

```diff
@@ -0,0 +1,227 @@
+<template>
+  <div class="filter">
+    <app-teleport to="#teleport-user-filter-toggle">
+      <app-filter-toggle
+        :active-filters-count="activeFiltersCount"
+        :expanded="expanded"
+        class="mr-1"
```

Comment:
We can delete this file right?

Reply:
This file has the logic to fetch for the initial list. But I can try to see if I can either delete it and move that logic to the main component or just remove what's inside it that is not needed

Reply:
Yeah we can just put the `doFetch` on the list-page or something